spamFilter.go

```diff
@@ -124,9 +143,23 @@ func isSpam(newMail *os.File, probMap map[string]float64) bool {
 		wordProbSlice[i].interest = math.Abs(0.5 - prob)
 		i++
 	}
-	quickSort(wordProbSlice, 0, len(wordProbSlice) - 1)
-	
-	
+	//quickSort(wordProbSlice, 0, len(wordProbSlice) - 1)
```

Comment:
When you decide to change approaches, you can leave the old approach behind in the git log instead of commenting it out.